spec/main-process/file-recovery-service.spec.js

```diff
@@ -0,0 +1,148 @@
+'use babel'
+
+import {BrowserWindow} from 'electron'
+import FileRecoveryService from '../../src/main-process/file-recovery-service'
+import temp from 'temp'
+import fs from 'fs-plus'
+import {Emitter} from 'event-kit'
+import sinon from 'sinon'
+
+describe("FileRecoveryService", () => {
+  let recoveryService, recoveryDirectory, windows, previousConsoleLog
+
+  function createWindow () {
```

Comment:
These could be extracted in a `spec-helper` probably, but I'd like to wait for more usages before doing so.